site/content/docs/commands/storage-init.md

```diff
@@ -3,7 +3,7 @@
 $ copilot storage init
 ```
 ## What does it do?
-`copilot storage init` creates a new storage resource attached to one of your services, accessible from inside your service container via a friendly environment variable. You can specify either *S3* or *DynamoDB* as the resource type.
+`copilot storage init` creates a new storage resource attached to one of your workloads, accessible from inside your service container via a friendly environment variable. You can specify either *S3*, *DynamoDB* or *Aurora* as the resource type.
```

Comment:
Do we explicitly introduce the term "workloads" anywhere in the docs? Maybe "...one of your workloads (services or jobs),..."? Then "...inside your workload container"?